File: consultoria/src/main/java/br/com/sigo/consultoria/dtos/DownloadContratoDTO.java
package br.com.sigo.consultoria.dtos;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class DownloadContratoDTO implements Serializable {

  private byte[] arquivo;

}

File: consultoria-app/app/src/main/java/br/com/sigo/consultoria/dtos/DownloadContratoDTO.kt
package br.com.sigo.consultoria.dtos

import java.io.Serializable

data class DownloadContratoDTO(var arquivo: ByteArray) : Serializable {
    override fun equals(other: Any?): Boolean {
        if (this === other) return true
        if (javaClass != other?.javaClass) return false

        other as DownloadContratoDTO

        if (!arquivo.contentEquals(other.arquivo)) return false

        return true
    }

    override fun hashCode(): Int {
        return arquivo.contentHashCode()
    }
}